common/ExpressApp.js

```diff
@@ -29,6 +30,7 @@ class ExpressApp {
     app.set('title', cfg.title || `Service Fabrik Broker (${type})`);
     if (cfg.ssl) {
       app.set('ssl', cfg.ssl);
+      app.use(helmet());
```

Comment:
@subhankarc ,  @kamath-prasad : I looked at this late. AFAIK, adding the helmet middleware is not enough to prevent HSTS attacks. One needs to also add code like the following as a peer dependency to `helmet` using the child `hsts` module:

```
app.use(helmet.hsts({
  maxAge: <time for expiry>
}))
```

Of course, this is from experience but from past experience with the `helmet` module, the idea was always to keep the module as a core and add dependencies as needed. Otherwise, the `helmet` module itself would become really heavyweight

Reply:
To add to this, from previous experience, adding the helmet middleware by default can also lead to side effects because a bunch of headers and processing happens on each request. This has led to issues like `max http header size exceeded` in some cases- just food for thought

Reply:
@abh1kg As far as I read, helmet by default adds maxAge as 1 year so there is no need to explicitly set it.

Reply:
By default, it adds the following

```
X-DNS-Prefetch-Control: off
X-Frame-Options: SAMEORIGIN
Strict-Transport-Security: max-age=15552000; includeSubDomains
X-Download-Options: noopen
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
```